templates/test/ci/prow-ci-version-windows-containerd-2022/patches/windows-image-update.yaml

```diff
@@ -12,5 +12,5 @@ spec:
         marketplace:
           publisher: cncf-upstream
           offer: capi-windows
-          sku: k8s-1dot23dot3-windows-2022-containerd
-          version: "latest"
+          sku: windows-2022-containerd-gen1
+          version: "124.1.20220608"
```

Comment:
what are your thoughts on just using version "latest" for these ci-version templates? That way we're always getting the latest image OS changes and since we don't actually use the k8s version on the image it doesn't matter which k8s version is used

Reply:
also if do that it would make sense to change the comment above from `an older image` to `the latest published image`

Reply:
we'd want to make this change for Linux as well if you want to change both in the same PR

Reply:
> what are your thoughts on just using version "latest" for these ci-version templates?

I did consider this and was hesitant, felt like it wasn't deterministic, would never know what image I got: could be 1.24 or 1.22 but I guess since we replace the Kubernetes bits in this case it doesn't matter to much. Prior to this I had a bit more control over the "latest" image since it would be the latest published for 1.23.3 image. Happy to try this route if you are and we can re-assess if it causes to many issues.